docs/docs/concepts/volumes.md

```diff
@@ -0,0 +1,143 @@
+# Volumes
+
+Volumes allow persisting data between runs. When you add a volume,
+`dstack` provisions a network disk in the cloud such as an AWS EBS.
+Then you can mount the volume as a directory in a run and store data there.
+After the run is terminated, the volume can be mounted again and the stored data will persist.
+
+`dstack` supports creating new volumes (a.k.a. `dstack`-managed volumes)
+and also registering existing volumes (a.k.a. external volumes).
+The latter allows accessing data that is already stored on some volume such as pre-processed training data.
+
+!!! info "Backends"
+    Currently, volumes are supported only for `aws`. Support for other backends is coming soon!
+
+!!! info "File system"
+    `dstack` creates an ext4 file system on `dstack`-managed volumes automatically.
+    If you register an external volume, you must ensure it already has a file system.
+
+## Creating new volumes
+
+First create a `volume` configuration file and specify `size` of the volume you'd like to provision:
+
+<div editor-title="new-volume.dstack.yml"> 
+
+```yaml
+type: volume
+name: my-new-volume
+backend: aws
+region: eu-central-1
+size: 100GB
+```
+
+</div>
+
+Then apply the configuration to create the volume:
```

Comment:
```suggestion
Then, apply the configuration to create the volume:
```

Reply:
It is permissible to avoid commas after short introductory phrases.